dashboard/app/views/scripts/show.html.haml

```diff
@@ -36,6 +36,21 @@
     .span12#rightside
       = render partial: 'shared/user_stats', locals: { user: current_user, width: '100%'}
 
+
+  -# Just some handy internal UI to show instruction text for all the levels in hoc2015.
+  - if current_user && current_user.admin? && rack_env == :development && @script.name == Script::HOC2015_NAME
```

Comment:
we could just make this admin all the time (no requirement on `:development` or `HOC2015_NAME`) - but you might want to get it reviewed by someone else like Laurel first...